RafRate.API/Controllers/EventController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using RafRate.Core.Services;
+
+namespace RafRate.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EventController : ControllerBase
```

Comment:
I think better to use "Holiday" not "Event", structure "Event" is already exist in C#
Please change it in other places